### PR TITLE
Test added apr7

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Added new & upadted 
[Test-Execution-logs-APR7.zip](https://github.com/project-chip/connectedhomeip/files/8442343/Test-Execution-logs-APR7.zip)
Manual Scripts:
Test_TC_BDX_1_4.yaml
Test_TC_BOOL_2_2.yaml
Test_TC_BOOL_3_1.yaml
Test_TC_BRAC_3_1.yaml
Test_TC_CC_2_2.yaml
Test_TC_CC_3_4.yaml
Test_TC_CC_4_5.yaml
Test_TC_CC_5_4.yaml
Test_TC_CC_6_4.yaml
Test_TC_CC_7_5.yaml
Test_TC_CC_9_4.yaml
Test_TC_DD_1_11.yaml
Test_TC_DD_1_12.yaml
Test_TC_DD_2_1.yaml
Test_TC_DD_2_2.yaml
Test_TC_DD_3_3.yaml
Test_TC_DD_3_4.yaml
Test_TC_DD_3_7.yaml
Test_TC_DD_3_8.yaml
Test_TC_DL_1_1.yaml
Test_TC_DL_1_2.yaml
Test_TC_DL_1_4.yaml
Test_TC_DL_1_5.yaml
Test_TC_DL_1_6.yaml
Test_TC_DL_1_7.yaml
Test_TC_DL_1_8.yaml
Test_TC_DL_1_10.yaml
Test_TC_DL_1_11.yaml
Test_TC_DL_1_13.yaml
Test_TC_DL_1_14.yaml
Test_TC_DL_1_15.yaml
Test_TC_DL_1_16.yaml
Test_TC_DL_1_17.yaml
Test_TC_DM_2_1.yaml
Test_TC_DM_2_3.yaml
Test_TC_GC_1_4.yaml
Test_TC_I_3_1.yaml
Test_TC_I_3_2.yaml
Test_TC_IDM_7_1.yaml
Test_TC_ILL_3_1.yaml
Test_TC_LC_1_1.yaml
Test_TC_LC_1_3.yaml
Test_TC_LC_1_4.yaml
Test_TC_LC_1_5.yaml
Test_TC_LO_1_1.yaml
Test_TC_LVL_2_3.yaml
Test_TC_MC_2_2.yaml
Test_TC_MC_10_7.yaml
Test_TC_MOD_1_3.yaml
Test_TC_NC_1_1.yaml
Test_TC_NC_1_2.yaml
Test_TC_NC_1_3.yaml
Test_TC_NC_1_4.yaml
Test_TC_NC_1_5.yaml
Test_TC_NC_1_6.yaml
Test_TC_NC_1_7.yaml
Test_TC_NC_1_8.yaml
Test_TC_NC_1_9.yaml
Test_TC_NC_1_10.yaml
Test_TC_NC_1_11.yaml
Test_TC_NC_1_12.yaml
Test_TC_NC_1_13.yaml
Test_TC_NC_1_14.yaml
Test_TC_NC_1_15.yaml
Test_TC_NC_1_16.yaml
Test_TC_NC_1_17.yaml
Test_TC_NC_1_18.yaml
Test_TC_OCC_3_2.yaml
Test_TC_OO_3_1.yaml
Test_TC_OO_3_2.yaml
Test_TC_PRS_3_1.yaml
Test_TC_PS_3_1.yaml
Test_TC_PSCFG_2_2.yaml
Test_TC_PSCFG_3_1.yaml
Test_TC_RH_3_1.yaml
Test_TC_SC_4_1.yaml
Test_TC_SC_4_3.yaml
Test_TC_SC_4_4.yaml
Test_TC_SC_4_6.yaml
Test_TC_SC_4_7.yaml
Test_TC_SC_4_8.yaml
Test_TC_SC_4_10.yaml
Test_TC_SWTCH_1_1.yaml
Test_TC_SWTCH_3_1.yaml
Test_TC_TM_3_1.yaml
Test_TC_TSTAT_3_1.yaml
Test_TC_TSTAT_3_2.yaml
Test_TC_TSUIC_3_1.yaml
Test_TC_WNCV_4_5.yaml
Test_TC_WNCV_5_1.yaml
Test_TC_WNCV_7_1.yaml

### CODE REVIEW

**Review by jmeg-sfy — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_BDX_1_4.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 28.1.4. [TC-BDX-1.4] Response to Receiver Initiated BDX Transfer session
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BDX_1_4.yaml:19]: This is probably not the right cluster?

File: src/app/tests/suites/certification/Test_TC_BOOL_2_2.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 68.2.2. [TC-BOOL-2.2] Primary functionality with server as DUT
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BOOL_2_2.yaml:19]: Definitely not the right cluster.

File: src/app/tests/suites/certification/Test_TC_BOOL_2_2.yaml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    endpoint: 0
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BOOL_2_2.yaml:20]: Not the right endpoint.

File: src/app/tests/suites/certification/Test_TC_BOOL_3_1.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 68.3.1. [TC-BOOL-3.1] Attributes with client as DUT
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic"
+    endpoint: 0
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BOOL_3_1.yaml:20]: Wrong cluster and endpoint?

File: src/app/tests/suites/certification/Test_TC_BRAC_3_1.yaml [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 75.3.1. [TC-BRAC-3.1] Global attributes with client as DUT
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic"
+    endpoint: 0
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BRAC_3_1.yaml:20]: Wrong cluster and endpoint?

File: src/app/tests/suites/certification/Test_TC_BRAC_3_1.yaml [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+          [1636523867.768810][6769:6774] CHIP:DMG:
+          [1636523867.768834][6769:6774] CHIP:DMG:        ],
+          [1636523867.768863][6769:6774] CHIP:DMG:
+          [1636523867.768885][6769:6774] CHIP:DMG: }
+          [1636523867.768962][6769:6774] CHIP:ZCL: ReadAttributesResponse:
+          [1636523867.768985][6769:6774] CHIP:ZCL:   ClusterId: 0x0000_0025
+          [1636523867.769007][6769:6774] CHIP:ZCL:   attributeId: 0x0000_FFFD
+          [1636523867.769024][6769:6774] CHIP:ZCL:   status: Success                (0x0000)
+          [1636523867.769045][6769:6774] CHIP:ZCL:   attribute TLV Type: 0x04
+      disabled: true
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BRAC_3_1.yaml:51]: Why is this disabled?  This should be automatable.

**Review by bzbarsky-apple — COMMENTED** [review]
Similar questions an all the test files.  Why do they all claim "Basic" for the cluster and disable even things that could be automatically tested?

**Comment by bzbarsky-apple** [comment]
So these files are script-generated?

In that case, can we modify the script to add a comment saying that the files are automatically generated and need careful auditing before being trusted or automated, and listing specific known shortcomings (the cluster/endpoint thing for sure, any others) that need to be addressed.

**Comment by raju-apple** [comment]
Yes , these are script generated. @kowsisoundhar12 @kvikrambhat  can you guys add the verbage which Boris is requesting and push the PR again please?

**Review by woody-apple — APPROVED** [review]

**Pull request merged**